<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>746</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: rgb(232, 232, 232);
}

QTableWidget {
    background-color: #fff;
    border: none;
    gridline-color: #ddd;
}
QHeaderView::section {
    background-color: #f2f2f2;
    font-weight: bold;
    border: none;
    padding: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1120</width>
     <height>746</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#tables, QFrame#stats {
    background-color: white;
    border-radius: 10px;
    padding: 10px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,11">
    <item>
     <layout class="QVBoxLayout" name="header_2">
      <item>
       <widget class="QFrame" name="header">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: rgb(85, 0, 0);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
	color: white;
	border: none;
	margin-left: 20px;
}</string>
           </property>
           <property name="text">
            <string>Student Information System</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="2,10">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QFrame" name="stats">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame #welcome{
    background-color: #fff;       
    border: 1px solid #ddd;       
    border-radius: 20px;    
    padding: 10px;             
}

QLabel{
	font-weight: bold;
	font-size: 20px;
}

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
           <item>
            <widget class="QLabel" name="studentCount">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="programCount">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="collegeCount">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="tables">
        <property name="styleSheet">
         <string notr="true">QHeaderView::section, QTableWidget::item {
    border-bottom: 1px solid rgb(85, 0, 0);
}

QHeaderView::section:last{
	color:transparent;
}

QTableWidget::item {
    background: white;
}

QTableWidget::item:selected {
    background-color: lightblue;
	color: rgb(85, 0, 0);
}

QTableWidget::item:hover {
    background-color: lightblue;
	color: rgb(85, 0, 0);
}

QTableWidget::row:hover {
    background-color: lightblue;
	color: rgb(85, 0, 0);
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,11">
         <item>
          <layout class="QHBoxLayout" name="functions" stretch="0,0,0,0,0,0">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QComboBox" name="sortBy">
             <item>
              <property name="text">
               <string>Sort By...</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="sortOrder">
             <item>
              <property name="text">
               <string>Order...</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ascending</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Descending</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="addButton">
             <item>
              <property name="text">
               <string>Add New...</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Add New Student</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Add New Program</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Add New College</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="searchBy">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::NoInsert</enum>
             </property>
             <item>
              <property name="text">
               <string>Search All</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="searchInput">
             <property name="placeholderText">
              <string>Search...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="styleSheet">
            <string notr="true">QTabBar::tab {
    background: white;
    border: 1px solid #ccc;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
    padding: 8px 15px;
}


QTabBar::tab:selected {
    background: #f5f5f5;
    border: 1px solid #ccc;
    border-bottom: none;
}

QTabWidget::pane {
    border: 1px solid #ccc;
    top: -1px;
}</string>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Students</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QTableWidget" name="studentsTable">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="locale">
                <locale language="Javanese" country="Indonesia"/>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOn</enum>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustToContents</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="textElideMode">
                <enum>Qt::ElideNone</enum>
               </property>
               <property name="showGrid">
                <bool>false</bool>
               </property>
               <property name="gridStyle">
                <enum>Qt::NoPen</enum>
               </property>
               <property name="columnCount">
                <number>7</number>
               </property>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>40</number>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <column/>
               <column/>
               <column/>
               <column/>
               <column/>
               <column/>
               <column/>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Programs</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QTableWidget" name="programsTable">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustToContents</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="showGrid">
                <bool>false</bool>
               </property>
               <property name="gridStyle">
                <enum>Qt::NoPen</enum>
               </property>
               <property name="columnCount">
                <number>4</number>
               </property>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>40</number>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <column/>
               <column/>
               <column/>
               <column/>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Colleges</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QTableWidget" name="collegesTable">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="locale">
                <locale language="Konkani" country="India"/>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustToContents</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="showGrid">
                <bool>false</bool>
               </property>
               <property name="gridStyle">
                <enum>Qt::NoPen</enum>
               </property>
               <property name="columnCount">
                <number>3</number>
               </property>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>150</number>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>40</number>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <column/>
               <column/>
               <column/>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
